index.js

```diff
@@ -222,4 +223,14 @@ module.exports = class BemEntityName {
     isEqual(entity) {
         return entity && (this.id === entity.id);
     }
+    /**
+     * Determines whether specified entity is instance of BemEntityName.
+     *
+     * @param {BemEntityName} entity - the entity to check.
+     *
+     * @returns {boolean} A Boolean indicating whether or not specified entity is instance of BemEntityName.
+     */
+    static isBemEntity(entity) {
+        return entity._isEntity;
```

Comment:
We can also look at `_data`, but this may change in one of the versions.

```js
return entity && entity._data && entity._data.block`;
```

I think your solution is more reliable.